### PR TITLE
MV3 Manifest keys updates

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary
Changes to the manifest keys for Manifest V3 support in the developer preview, including:

- update to the keys list on the overview page to add details of supported manifest versions
- addition of a "Manifest version" line into the overview table of every manifest key page
- addition of the `action` key (as a copy of the `browser_action` key) with cross-reference information between the two
- update to `favicon_url` of the  [chrome_settings_overrides](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/chrome_settings_overrides) manifest key that a local URL must be provided in MV3.  

#### Metadata

This PR…

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ browser-compat: webextensions.manifest.content_security_policy
       <th scope="row">Mandatory</th>
       <td>No</td>
     </tr>
+    <tr>
+      <th scope="row">Manifest version</th>
+      <td>2 or higher</td>
```
Comment (mixedpuppy) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:24]: probably doesn't matter here, but do we note it is different in 3?

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/offline_enabled/index.md [hunk 2]
```diff
@@ -23,6 +23,10 @@ browser-compat: webextensions.manifest.offline_enabled
       <th scope="row">Mandatory</th>
       <td>No</td>
     </tr>
+    <tr>
+      <th scope="row">Manifest version</th>
+      <td>2 or higher</td>
```
Comment (mixedpuppy) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/offline_enabled/index.md:28]: This is unsupported in Fx, I've no idea if chrome supports it in 3.

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/user_scripts/index.md [hunk 3]
```diff
@@ -21,6 +21,10 @@ browser-compat: webextensions.manifest.user_scripts
       <th scope="row">Mandatory</th>
       <td>No</td>
     </tr>
+    <tr>
+      <th scope="row">Manifest version</th>
+      <td>2 or higher</td>
```
Comment (mixedpuppy) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/user_scripts/index.md:26]: only in 2, for mv3 see scripting api (user scripts is not yet moved to the scripting api, but access to user scripts is removed in mv3)

**Review by mixedpuppy — APPROVED** [review]
r+ with comments addressed (only one is necessary, user scripts)

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md (hunk 1 above)
Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:24]: Next on my list to look at updating the content security policy documentation

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/offline_enabled/index.md (hunk 2 above)
Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/offline_enabled/index.md:28]: The browser compatibility data aligns with this being unsupported in Firefox. Still listed as a supported item on the Chrome Manifest V3 [Manifest file format](https://developer.chrome.com/docs/extensions/mv3/manifest/) page, however (unlike the similar listing on the V2 Manifest file format) there isn't a detail page describing the key (error when following link). Any idea who to check with to confirm?

Comment (mixedpuppy) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/offline_enabled/index.md:28]: I'd just follow what is here: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/offline_enabled

Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/offline_enabled/index.md:28]: OK, I assumed the broken link in Chrome docs means that the inclusion in their V3 manifest layout was an error and it's not supported. Updated.

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/user_scripts/index.md (hunk 3 above)
Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/user_scripts/index.md]: Done

**Pull request merged**